app/models.py

```diff
@@ -2676,6 +2676,27 @@ class UnsubscribeRequestReport(db.Model):
     processed_by_service_at = db.Column(db.DateTime, nullable=True)
     count = db.Column(db.BigInteger, nullable=False)
 
+    def serialize(self):
+        return {
+            "batch_id": str(self.id),
+            "count": self.count,
+            "earliest_timestamp": self.earliest_timestamp.isoformat(),
+            "latest_timestamp": self.latest_timestamp.isoformat(),
+            "processed_by_service_at": self.processed_by_service_at.isoformat(),
```

Comment:
This would raise an error for an unprocessed `unsubscribe_request_report` which would have a `latest_timestamp` of `None`.
Maybe something along the lines of:
`"processed_by_service_at":
                self.processed_by_service_at.isoformat() if self.processed_by_service_at else None`.

It would also mean adding test coverage for an `unsubscribe_request_report` with no `processed_by_service_at` value.

Reply:
Good spot - have added this in 2b8d9ea